estate_account/models/estate_property.py

```diff
@@ -0,0 +1,27 @@
+from odoo import models, Command
+
+class estate_property(models.Model):
+    _inherit = 'estate.property'
+
+    def sold_property_button(self):
+        res = super().sold_property_button()
```

Comment:
Your `sold_property_button` in estate does not return anything. Thus, this one should not either
```suggestion
        super().sold_property_button()
```